lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -301,8 +302,9 @@ runGhcid
   -> [CabalPackageInfo]
   -> Maybe String -- ^ Optional command to run at every reload
   -> m ()
-runGhcid dotGhci packages mcmd = withProjectRoot "." $ \root ->
-  nixShellWithPkgs root True (packageInfoToNamePathMap packages) (Just $ unwords $ ghcidExePath : opts) -- TODO: Shell escape
+runGhcid dotGhci packages mcmd = withProjectRoot "." $ \root -> do
+  rootAbs <- liftIO $ makeAbsolute root
+  nixShellWithPkgs rootAbs True (packageInfoToNamePathMap packages) (Just $ unwords $ ghcidExePath : opts) -- TODO: Shell escape
```

Comment:
Should this `makeAbsolute`ing just be done by `nixShellWithPkgs`?

Reply:
Quite possibly

Reply:
Go ahead and do that...